### ISSUE TITLE
Add tests for BatchNorm backward in evaluate mode

### ISSUE THREAD

**Issue by apaszke** [pytorch/pytorch#590]
Just so we don't forget (#589).
